### ISSUE TITLE
Change the default emoji list

### ISSUE THREAD

**Issue by borisrunakov** [vector-im/element-android#5638]
#### What would you like to do?
It would be nice to be able to configure the default emojis that appear, on long pressing on a message. Currently the emojis that appear by default are : :+1: :-1: :smile: :tada: :confused: :heart: :rocket: :eyes: 
#### Why would you like to do it?
Some users coming from other messaging apps are used to other emojis and picking the one they want in Element Android is only possible by, long pressing > Add Reaction > pick emoji . Facebook messenger for example is using :heart: :laughing: :open_mouth: :cry: :angry: :+1: 
#### How would you like to achieve it?
A preferences menu could be added to select the default emojis a user wants.

**Comment by apsarras** [comment]
YES! I mostly use reactions that are not in the default list. Whenever I want to post such a reaction, I need to:
1. Long-tap message
2. Tap Add Reaction...
3. Tap "back" because the keyboard popped up for whatever reason and hides the emoji I'm looking for
4. Scroll to the right emoji and tap

Such a feature would save me steps 2, 3, 4 80% of the time

**Comment by nikdelig** [comment]
Same here, takes too long to respond to messages with an emoji different than the ones from the default list

**Comment by bmarty** [comment]
The list comes from GitHub:

<img width="183" alt="image" src="https://user-images.githubusercontent.com/3940906/161225321-b2a587a5-832b-4eb6-9df5-5b518a6f9276.png">

What we could do is adding a list of frequently used reaction, as per EleWeb:

<img width="341" alt="image" src="https://user-images.githubusercontent.com/3940906/161225473-e4668e0d-b4ca-495d-8c0e-4e9ddf65683d.png">

**Comment by borisrunakov** [comment]
You mean by long pressing a message > `Add Reaction` > present the `Frequently Used` list ? If that's so, the user needs an extra step to add a reaction in contrast to what other messaging apps (like fb messenger) offer. 
It's better than nothing, but the goal is to quickly add a reaction as easy as possible.

**Comment by chagai95** [comment]
It seems like https://github.com/vector-im/element-android/issues/871 is for adding such a list to Android? Regarding implementation: is this done in the account data? Would it make sense to do that there? Maybe it makes sense to do this in account room data, so I can see what emoji I use most with what room? ;) I think this could also be cool so that I can add some variety if I notice that I'm kind of always using the same emojis with some people...

**Comment by immanuelfodor** [comment]
I would really like to replace, for example, the 😄 reaction with 😆 and tehere is no way to do it currently in the app. Long pressing the reactions to replace them would be fine for me. Or the app could "learn" the last used reactions and update the list on its own based on usage frequency or last usage. As an MVP, a long press to replace could work just fine.

**Comment by Fmstrat** [comment]
The way Signal handles this where you can configure your defaults is perfect. I would love that in Element.
